### DISCUSSION TITLE
Fox Movies | (Slovenia)

### DISCUSSION THREAD

**Discussion by R4IN2007** [iptv-org/community#299] [Show and tell]
### Channel Name

Fox Movies

### Channel ID

FoxMoviesSlovenia.si

### Stream URL

http://194.163.179.246/slovenci/filmski/foxmovies.m3u8

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
<img width="678" alt="image" src="https://user-images.githubusercontent.com/7253922/185268069-97514f43-8f88-48a8-a2a2-188562d62f68.png">

**Comment by AntiPontifex** [comment]
Dead issue: please close.
